pyproject.toml

```diff
@@ -14,6 +14,9 @@ license = "Apache-2.0"
 
 packages = [{ include="cognite", from="." }]
 
+[tool.ruff]
+line-length = 120
```

Comment:
Objections to this, or any reason why ruff isn't configured here? My IDE explodes the files if it doesn't find any line length config in pyproject. I can work around it, but it's convenient to avoid that...

Reply:
It's defined in `.pre-commit-config.yaml`

Reply:
I'm aware, but this is more general than defining it as an argument in pre-commit.

Reply:
Alright. I really think that's just wrong, but I'll remove it.